### ISSUE TITLE
Allowing users to filter posts by user

### ISSUE THREAD

**Issue by nikilselvam** [psybuzz/goldenconversations#51]
Hey guys, I was talking to one of my friends about this project and about conversation threads in general, and he mentioned something that struck me. He said that when he is having conversations on Facebook, he oftentimes wishes he could filter the conversation posts by user. This is very hard to do when there are lots and lots of posts in conversation, and it can require lots of scrolling in order to find one specific post that you may be looking for.

I think this is a cool feature that we should think about implementing at some point. It could be cool to go back and view a conversation from only one person's viewpoint, particularly since our site focuses so much on what each individual brings to a conversation.  What do you guys think?

**Comment by psybuzz** [comment]
I think filtering posts selectively by user could be a neat way to interact with a discourse.  We probably have enough tasks that should be handled before the beta release, but I'd like to see how these filters work in action after the release.

Still, I wonder if it would make sense to read "I agree with your point", or "That which you mentioned...", in a post without seeing the content it was referring to.  It will be interesting to see how useful this feature will be.

**Comment by tonyxj** [comment]
Yes, this could be a neat feature. I think this is one where its usefulness perhaps would only show after the user has create a bunch(a crunch) of conversations, I'd say more than perhaps 8 to 10. As when the app only a few convos, the names can be seen at a glance relatively clearly. I think we put this in the icebox and come back to after users have been created conversations for a while.

**Comment by nikilselvam** [comment]
Sounds good! I think this will be a cool feature to tackle down the line.
